app/src/main/res/layout/profile_progress_fragment.xml

```diff
@@ -24,11 +24,11 @@
         android:id="@+id/profile_progress_list"
```

Comment:
At top level `ConstraintLayout` add `android:background="@color/white"`. It is available in tablet files but not in mobile files. But ideally it should be available for all files.

Once this change is done please recreate after screenshots and update PR description accordingly.